app/views/shared/_asia_fields.html.slim

```diff
@@ -35,22 +34,26 @@
                   label: false,
                   collection: Asia.collections[:neurological_motor_level_right]
       .col-md-4
+        = wrapped_form_for(@patient) do |f|
```

Comment:
This is what's causing the form submit to break.  wraped_form_for creates another <form> element within the embedding <form> and confuses all the later code.

If we have to do this, we should pass in the parent form builder as another variable other than |f|, however I'm a bit confused at the UI here.  Should the `sci_type` be per patient, or per Asia evaluation? If it's per patient, having it on the asia widget is confusing...